assets/component-card.css

```diff
@@ -81,6 +99,11 @@
   margin-top: var(--collapsed-shadow-offset-y);
 }
 
+.card.card--card .card__inner .card__media {
+  border-bottom-right-radius: 0;
+  border-bottom-left-radius: 0;
+}
+
 .card--standard.card--text {
   background-color: transparent;
 }
```

Comment:
NOt sure if we care about this scenario since it's for standard cards with horizontal spacing and no vertical one: 
![](https://screenshot.click/22-23-fj3k1-61e0l.png)

Same here: [screenshot](https://screenshot.click/22-21-om5go-z4vw2.png). But im pretty sure we're good 👍